docroot/themes/custom/yukonca_glider/src/sass/base/_base.scss

```diff
@@ -1,12 +1,43 @@
 @layer base {
+  *,
+  ::before,
+  ::after {
+    border-width: 0;
+    border-style: solid;
+  }
+
   html,
   body {
     @apply min-h-full font-base text-black;
   }
 
+  .field__content {
```

Comment:
Are you sure that we need to add these rules to all pages? 
This is a question to all changes in this file.